boxoffice/messageclient.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+from baseframe import _
+
+from . import app, rq
+from .models import Order
+
+
+@rq.job('boxoffice')
+def send_telegram_message(order_id):
+    with app.test_request_context():
+        order = Order.query.get(order_id)
+        message_text = _("{user} purchased {title}").format(
+            user=order.buyer_fullname, title=order.line_item.item.title
+        )
```

Comment:
Model `Order` has no attribute named `line_item`. It has a list named `line_items`. You'll have to get the title from each item in the list and construct a larger message with the full order.